### PR TITLE
Changes made to make submit etc different

### PR DESCRIPTION
**Pull request by khenriksson**
All my small changes regarding the design with the submit button and skip button looking different etc etc.

### CODE REVIEW

**Review by VonLatvala — CHANGES_REQUESTED** [review]
Scrollbars. Sec......

**Review by VonLatvala — DISMISSED** [review]
Must say that I'm not a huge fan of the feint colors in select buttons (the ones you can toggle). But will leave the outlook design to you. When #132 is merged into this, I approve the changes.

File: src/components/FreeText.jsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default class FreeText extends React.Component {
                         rows="10"
                         margin="normal"
                         value={this.props.text}
+                        placeholder="Open feedback"
```
Comment (andersnylund) [inline src/components/FreeText.jsx]: Shouldn't this placeholder be `<FormattedMessage id="textfield.placeholder" />` ?

**Review by andersnylund — CHANGES_REQUESTED** [review]

**Review by jallusparta — CHANGES_REQUESTED** [review]
The multi select buttons shouldn't get wider when selected.

**Review by jallusparta — DISMISSED** [review]
Näyttökuva 2019-4-10 kello 15 40 23

The placement of the text buttons to the middle, and some select buttons, doesn't work.

**Comment by jallusparta** [comment]
> Must say that I'm not a huge fan of the feint colors in select buttons (the ones you can toggle). But will leave the outlook design to you. When #132 is merged into this, I approve the changes.

I agree with Axel on the colors of the multiselect buttons. I think send is okay neutral, but skip buttons color is too much of a different tone.

**Comment by tomrailio** [comment]
4 shades of blue. Perhaps one too many.

**Comment by khenriksson** [comment]
Then do you want me to change it to a completely different color? It's quite hard to find something that works with our other blue colors.

File: src/components/FreeText.jsx (hunk 1 above)
Comment (khenriksson) [reply src/components/FreeText.jsx]: Not sure, changed it to that but gave me this
image

**Review by VonLatvala — APPROVED** [review]
Looks good, can't see finnish or swedish questions (application crash). But these changes don't introduce this problem.

**Review by andersnylund — APPROVED** [review]
Same as what @VonLatvala said, we have to fix the change language bug in develop, and preferably also #134, before merging

**Pull request merged**